Makefile

```diff
@@ -24,6 +24,7 @@ build-cli:
 .PHONY: build-plugins
 build-plugins: 
 	go build -o ./bin/plugins/ ./plugins/verifier/cosign
+	go build -o ./bin/plugins/ ./plugins/verifier/licensechecker
```

Comment:
Should this be `spdx-{type}` in this case `spdx-tools`  where type is the underlying implementation details like clearly defined or another project we use?

Reply:
I have updated to spdx-tools for now. We can make updates if necessary in future iterrations.